src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -83,6 +85,18 @@ export default class MeetingContainer extends Component {
     }
   }
 
+  componentWillUnmount() {
+    // Time for clean up!
+    this.resetContainer();
+  }
+
+  resetContainer() {
+    const {dispatch, teamSub, memberSub} = this.props;
+    teamSub.unsubscribe();
```

Comment:
yeah, my vision is to unsub on unmount: https://github.com/ParabolInc/action/issues/150#issue-169170390

already, redux-socket-cluster will unsub on dismount, but the trick is only unsubbing from unshared subs. For example, switching from teamDash to userDash we want to keep the `teamMembers/:teamId` subscription going. 

I'm not sure what the right pattern is. Right now, redux-socket-cluster is handling subs at the socket layer (it unsubs from everything when that component gets unmounted). cashay is handling subs at the sub layer (it unsubs from a socket when you call unsubscribe). Part of me thinks this is right, we should ignore the socket layer & just unsub using cashay all the time.